ghengin/ghengin-geometry/Ghengin/Geometry/Cube.hs

```diff
@@ -22,12 +22,13 @@ import Ghengin.Geometry.Transform
 -- | A unit cube centered at (0,0,0).
 cube :: [Vertex '[Vec3]]
 cube = concatMap (`transform` square)
-  [                    translateV (vec3     0      0 (-0.5)) -- front face
-  , rotateY pi      <> translateV (vec3     0      0   0.5 ) -- back face
-  , rotateY (pi/2)  <> translateV (vec3   0.5      0     0 ) -- right face
-  , rotateY (-pi/2) <> translateV (vec3 (-0.5)     0     0 ) -- left face
-  , rotateX (-pi/2) <> translateV (vec3     0    0.5     0 ) -- top face
-  , rotateX (pi/2)  <> translateV (vec3     0  (-0.5)    0 ) -- bottom face
+                                                             -- TODO: These labels might be wrong:
+  [ translateV (vec3     0      0 (-0.5))                    -- front face
+  , translateV (vec3     0      0   0.5 ) <> rotateY (-pi)   -- back face
```

Comment:
Note that the rotations have all been reversed. That's probably not the right way to do it?

Reply:
According to my understanding of what @dpwiz said, this does seem correct.

Reply:
Translating by -0.5 will give you a "back" face if you view along +Z. Since it will move it behind the camera at 0,0,0.

I assume those are world-space points, so they wouldn't be affected by the reverse-z.

Reply:
`pi` or `-pi` wouldn't differ here - you'll do a 180 anyway.
The signs are ofc important for the quarter turns.
top/bottom angles seem correct, but I expect left/right to be flipped. Negative turn from the front is "left".

Reply:
Nevermind. Both in `simple-cube` and `simple-camera` we use a handmade cube.

I don't think the `ghengin-geometry` cube is being used anywhere at the moment 👍

Reply:
Doing transforms in clipspace is sus 😅

Reply:
Did I understand correctly, @dpwiz, that where we previously had `A <> B` now we need to have `B <> A` because you've inverted the order of the operation?

If so, I would expect these changes in `Cube.hs` to be necessary. Perhaps I didn't get it.